### ISSUE TITLE
responsive design and better UX with CTA button

### ISSUE THREAD

**Issue by pMattheew** [costaluu/task-it-easy#5]
I browse the internet in my laptop and it has a small screen, making the application interface going unconvenient in proportions, having big titles and small buttons; this _can be_ a bad UX.

![image](https://user-images.githubusercontent.com/99044724/233818425-6cfe2b44-199d-48b0-8969-45d6789ce874.png)

also, when I first accessed the app from the [first link](https://task-it-easy.vercel.app/) that you gave (I saw in tabnews), it doesn't give direct access to the application (as the natural instinct awaits it to be), but to the docs, and this isn't bad, the bad is that in the landing page there's not a CTA button redirecting to the application, only a tiny `App >` button in navbar. I had to search in it for how to run the app and only after reading a paragraph saying about the tiny button I realized it. This could be a new separate issue.

btw very beautiful app, congrats doing it; it can be seen that you had some work with the theme feat and others, you're in the right way 👏🏾

**Comment by costaluu** [comment]
Hey glad you liked!

> I browse the internet in my laptop and it has a small screen, making the application interface going unconvenient in proportions, having big titles and small buttons; this can be a bad UX.

Yes, I got you, the [app](https://app-task-it-easy.vercel.app/) came from the old version of the app. This page in specific page will became just a "create a session" page just like [Notion](https://www.notion.so/pt-br).

> also, when I first accessed the app from the [first link](https://task-it-easy.vercel.app/) that you gave (I saw in tabnews), it doesn't give direct access to the application (as the natural instinct awaits it to be), but to the docs, and this isn't bad, the bad is that in the landing page there's not a CTA button redirecting to the application, only a tiny App > button in navbar. I had to search in it for how to run the app and only after reading a paragraph saying about the tiny button I realized it. This could be a new separate issue.

About the CTA, you're right I didn't give it enought attention, in the near future this "web version" will became a desktop/mobile versions.

[ ] - CTA button to the web app.

**Comment by costaluu** [comment]
I think it's cleaner now ;)

![image](https://user-images.githubusercontent.com/44981765/234299479-06df3924-729a-45bf-8194-82dccbfd4e77.png)

**Comment by pMattheew** [comment]
whoa very cool, glad you liked the suggestions! Now the appearance is cleaner and the user is really focused in creating a session haha
